tests/loginAndRegister.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { Page, expect, test } from "@playwright/test"
+import {PageManager} from "../page-objects/pageManager"
+
+
+test.describe('registration', () =>{
+    test.beforeEach( async({page}) =>{
+        //Instance of Page Manager Class
+        const pm = new PageManager(page)
```

Comment:
Object of the PageManager (pm), can be initialised out of beforeEach and test, so that we can use one object in both cases.